lib/LANraragi/Model/Config.pm

```diff
@@ -40,6 +40,10 @@ sub get_redis {
         reconnect => 3
     );
 
+    if ($ENV{BREWMODE}) {
+        $redis->config_set("dir", $ENV{HOME} . "/Library/Application Support/LANraragi/");
```

Comment:
The other overrides are fine since they're modifying application defaults, but here we're modifying redis settings, which is something I've typically been doing through a custom configuration file: You can see an example [here](https://github.com/Difegue/LANraragi/blob/dev/tools/DockerSetup/redis.conf) for the Docker build.  

I don't know how easy it is to sideload conf files in Homebrew, but I feel it'd make a better approach than hard-setting it in code.  

The redis.conf file also redirects the redis logs to the LRR log directory, which is something you could be doing here as well -- It makes the redis logs available in the app which is always a plus.

Reply:
Also a good point, (I think) I implemented that behavior!